www/css/style.css

```diff
@@ -1746,23 +1748,23 @@ span.rz-bar-wrapper {
   border-top-right-radius: 5px;
   border-bottom-right-radius: 5px;
   background-color: white;
-  color: #01D0A7;
+  color: #C3E669;
 }
 .unit-toggle-right-active {
   width: 50%;
   float: right;
   height: 35px;
   border-top-right-radius: 5px;
   border-bottom-right-radius: 5px;
-  background-color: #01D0A7;
+  background-color: #C3E669;
   color: white;
 }
 .userdata-title {
   color: #ccc;
 }
 .carbon-goal-chart {
   height: 5px;
-  background-color: #01D0A7;
+  background-color: #C3E669;
```

Comment:
looks like it could have been cooler, though. Duplicating so many times was a gbad design choice on the part of the initial authors